platform/javascript/export/export.cpp

```diff
@@ -536,23 +561,23 @@ Error EditorExportPlatformJavaScript::run(const Ref<EditorExportPreset> &p_prese
 		return OK;
 	}
 
-	String basepath = EditorSettings::get_singleton()->get_cache_dir().plus_file("tmp_js_export");
-	String path = basepath + ".html";
-	Error err = export_project(p_preset, true, path, p_debug_flags);
+	const String basepath = EditorSettings::get_singleton()->get_cache_dir();
+	Error err = export_project(p_preset, true, basepath.plus_file("tmp_js_export.html"), p_debug_flags);
 	if (err != OK) {
 		// Export generates several files, clean them up on failure.
-		DirAccess::remove_file_or_error(basepath + ".html");
-		DirAccess::remove_file_or_error(basepath + ".js");
-		DirAccess::remove_file_or_error(basepath + ".pck");
-		DirAccess::remove_file_or_error(basepath + ".png");
-		DirAccess::remove_file_or_error(basepath + ".wasm");
+		DirAccess::remove_file_or_error(basepath.plus_file("tmp_js_export.html"));
+		DirAccess::remove_file_or_error(basepath.plus_file("tmp_js_export.js"));
+		DirAccess::remove_file_or_error(basepath.plus_file("tmp_js_export.pck"));
+		DirAccess::remove_file_or_error(basepath.plus_file("tmp_js_export.png"));
+		DirAccess::remove_file_or_error(basepath.plus_file("tmp_js_export.wasm"));
+		DirAccess::remove_file_or_error(basepath.plus_file("favicon.png"));
 		return err;
 	}
 
-	IP_Address bind_ip;
-	uint16_t bind_port = EDITOR_GET("export/web/http_port");
+	const uint16_t bind_port = EDITOR_GET("export/web/http_port");
 	// Resolve host if needed.
-	String bind_host = EDITOR_GET("export/web/http_host");
+	const String bind_host = EDITOR_GET("export/web/http_host");
+	IP_Address bind_ip;
```

Comment:
What's the benefit of `const` here? Why the re-ordering? This seems just noise :(

Reply:
> What's the benefit of const here? Why the re-ordering? This seems just noise :(

I've always added `const` when possible in my PRs. It makes "immutable" variables more obvious, which is generally favored these days.

As for the reordering, my intent was to put the variable declaration as close as possible to the first use of the variable (which generally helps with readability too).

Reply:
Fair enough